storybook/sidebar.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line @typescript-eslint/no-restricted-imports
+import { createElement, useMemo } from 'react';
+import { useStorybookApi } from '@storybook/manager-api';
+import { styled } from '@storybook/theming';
+
+/**
+ * Internal dependencies
+ */
+import badges from './badges';
+
+const STEM = 'sb-';
```

Comment:
What was the intent for this prefix? I'm guessing it's short for "sidebar"? (At first I thought it was for "storybook" 😆)

If we assume `tags` is going to be our source of truth for multiple categorizations, to be used in multiple ways (badges, sidebar icons, maybe filtering, etc), a more useful prefix here could be something like `status-`. And eventually when Storybook adds a filter-by-tag feature, we might add "component type" tags, like `type-form` or `type-layout` or whatever.

Reply:
Switched it to `status-`, for the reasons given (d450598f70bb8be90fc415943850b1b7c6f47c05)